### ISSUE TITLE
[v3.3.0-beta17]rsync server down? Never finds update.

### ISSUE THREAD

**Issue by czyz** [recalbox/recalbox-os#790]
For what it's worth - I did post to IRC first a few times, as suggested in this form. But it seems to be a ghost town over there.

### What happened
I'd like to update v3.3.0-beta17 to v4.0.0-beta1. When I choose 'Start UPDATE' from the system menu I'm told there's "NO UPDATE AVAILABLE".

When I ssh in and run rsync-update.sh, it reports: "failed to connect to recalbox.com (217.70.184.38): No route to host (113)". Yet I can ping recalbox.com from that same command line without issue.

Thanks for any help or information.

**Comment by menelkir** [comment]
I can't be wrong, but perhaps updates from "stable" to a "beta" version are
blocked for security/stability sake.

**Comment by menelkir** [comment]
I mean.. From 3.x to 4.x, if you know what I mean.

**Comment by rockaddicted** [comment]
Almost people of this project are living in France... So we are on the `CET (Central European Time)
UTC/GMT +1 hour` time zone...
So sorry to be in my bed at 4am when you come to ask a question in the "ghost town" called irc...
`2016-03-11 04:16:08	-->	czyzczyz (xxx) a rejoint #recalbox`

**Comment by substring** [comment]
@czyz @menelkir When submitting an issue, there is atemplate asking users to :
Be sure your issue is not already reported : check https://github.com/recalbox/recalbox-os/wiki/Known-Bugs-%28EN%29
And there, it's clearly stated that **you can't upgrade from 3.X.X to 4.0.0, you need a fresh install**

**Comment by czyz** [comment]
Apologies for sounding impertinent about IRC. I was just meaning to note that I did try to follow protocol. Amusingly, my first post to IRC was an apology for not knowing French and for writing in English, and the only response was from someone teasing me for my apology. I then came back to the channel a couple of times later to post the problem again but, as you note, everyone who could answer was likely asleep. I didn't expect that all developers or advanced users of the project were in the one timezone or a narrow range of zones, Recalbox is such a great project that I figured there'd likely be a more global spread of participants in the channel.

I did search github and did find several issues of people having trouble with updates, and in each case there was a response along the lines of "oh, the server's down, I'll restart it." I thought that might be the case with my situation since the update script was stating that rsync couldn't connect to the server, and I figured that posting here would allow the controller of the rsync server to see the message at their leisure rather than having to be logged in and watching the irc channel at the moment of my question.

I wish I'd found my way to the known bugs for v4.0.0 page, but I probably didn't do so because I was experiencing a problem in a v3x release and wouldn't have thought that v4x issues would affect me until I was running that version. I will do a fresh install.

Sorry for the misunderstanding.
